src/protocolstatus.cpp

```diff
@@ -129,9 +129,6 @@ void ProtocolStatus::sendStatusString()
 	map.append_attribute("width") = std::to_string(mapWidth).c_str();
 	map.append_attribute("height") = std::to_string(mapHeight).c_str();
 
-	pugi::xml_node motd = tsqp.append_child("motd");
-	motd.text() = g_config.getString(ConfigManager::MOTD).c_str();
-
```

Comment:
I'd recommend at least returning an empty string here, othewise you might break server lists like otservlist or such.